templates/zerver/app/index.html

```diff
@@ -132,6 +132,7 @@ <h3>{{ _('Loading...') }}</h3>
                 </div>
                 {% endif %}
             </div><!--/tab-content-->
+            {% include "zerver/app/home.html" %}
```

Comment:
Can you explain what's going on with this move?  It seems weird to be moving `html.html` outside `#home` .

Reply:
that move was a way to make it work.  (something about a  `overflow` setting of a higher `div`, but I don't really get it though).

However I just noticed it can also be fixed differently, by  removing  `tab-content` from `class="column-middle-inner"` , 
I guess it is there for a reason, but not sure why exactly.

Reply:
`position:sticky` looks at a parent div to calculate its position. `tab-content` has `overflow:auto` and therefore the calculation fails. By placing it outside `tab-content` it is able to get the height of the middle column.

Adjusting/removing this opens a whole can of worms and requires a lot of changes in positioning of many elements.